src/main/java/org/radarbase/management/web/rest/SubjectResource.java

```diff
@@ -273,18 +275,9 @@
                     .findAllByExternalIdAndAuthoritiesIn(externalId, authoritiesToInclude);
```

Comment:
shouldn't this be included in the Subject filter? what happens if some want to query subjects who are inactive using additional parameters?

Reply:
I originally left this line as-is since this branch uses a different SubjectMapper method.

Updated for consistency, now it uses the Subject Filter as well and supports pagination.